governance/votes/Executive Vote - August 2, 2023.md

```diff
@@ -0,0 +1,141 @@
+| opensky          | [0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04](https://etherscan.io/address/0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04)  | Composable          | 14.9         |
+| David Phelps     | [0xd56e3E325133EFEd6B1687C88571b8a91e517ab0](https://etherscan.io/address/0xd56e3E325133EFEd6B1687C88571b8a91e517ab0)  | Composable          | 8.94         |
+| seedlatam.eth    | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d)  | Sovereign Finance   | 11.92        |
+| StableLab        | [0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08](https://etherscan.io/address/0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08)  | Growth              | 14.9         |
+| flipsidegov      | [0x300901243d6CB2E74c10f8aB4cc89a39cC222a29](https://etherscan.io/address/0x300901243d6cb2e74c10f8ab4cc89a39cc222a29)  | Growth              | 14.9         |
+
+This comes to a total of **134.1 MKR**.
+
+### DAO Resolution
+
+As per this forum [post](https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291) that has been approved by the Stability Scope Facilitators, the following DAO Resolution will be approved if this executive proposal passes.
+
+- Approve the DAO Resolution with the IPFS hash `QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p` - the DAO Resolution may be viewed [here](https://gateway.pinata.cloud/ipfs/QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p).
+
+In summary, this will instruct the trustees of the Monetalis Clydesdale (RWA007-A) JAT1 arrangement to sell a portion of ETF holdings and return profits to MakerDAO via the input conduit.
+
+### Launch Project Funding
+
+As per this forum [post]($TBD) from the Accessibility Scope Facilitators, as per the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget), the following DAI transfer to support the Launch Project will take place, if this executive proposal passes.
+
+- Transfer **2,000,000 DAI** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+
+### Spark Proxy Spell
+
+As part of this executive proposal, a Proxy Spell for Spark Protocol at [0x443f3f4328553f5f85dFc0BA3D59969708201E14](https://etherscan.io/address/0x443f3f4328553f5f85dFc0BA3D59969708201E14) will be triggered if this executive proposal passes.
+
+This will include the following changes to Spark Protocol:
+
+#### DAI InterestRateStrategy Update
+
+- Deploy a new DAI_interestRateStrategy to tie the borrow rate of Dai to the ETH-C Stability Fee. This is required due to the activation of the EDSR, as vault stability fees and other rates will continue to be based on the underlying DSR rather than the EDSR. The ETH-C stability fee equals the underlying DSR, making it an appropriate reference for Spark Protocol.
```

Comment:
```suggestion
Deploy a new DAI_interestRateStrategy to tie the borrow rate of Dai to the ETH-C Stability Fee. This is required due to the activation of the EDSR, as vault stability fees and other rates will continue to be based on the underlying DSR rather than the EDSR. The ETH-C stability fee equals the underlying DSR, making it an appropriate reference for Spark Protocol.
```

Is the new DAI strategy actually deployed as part of the Proxy Spell?
Maybe link to Stability Fee docs again.

Not sure the explanation as written is going to help anyone that doesn't already know why it's happening, to be honest. It's moderately difficult to explain, admittedly.